packages/editor/src/client/panel/imageEditPanel.tsx

```diff
@@ -302,11 +265,22 @@ export function ImagedEditPanel({id, file, onClose, onSave}: ImageEditPanelProps
                 <FormGroup>
                   <ControlLabel>{t('images.panels.description')}</ControlLabel>
                   <FormControl
+                    rows={5}
+                    name="textarea"
+                    componentClass="textarea"
                     value={description}
                     disabled={isDisabled}
                     onChange={value => setDescription(value)}
                   />
                 </FormGroup>
+                <FormGroup>
```

Comment:
Is this a request that users can change file names? I think it's rather confusing having an image name and a file name. I'd prefer it if we don't allow this to change and also don't display it so prominently.